client/controller.js

```diff
@@ -0,0 +1,83 @@
+import { Client } from './classes/client.js';
+import { ManualTranslator } from './classes/manualTranslator.js';
+import { Simulator } from './classes/simulator.js';
+import { languages as LANGUAGECODES } from './data/languageCodes.js';
+import { textList as TEXTLIST } from './data/sampleTexts.js';
+const EL = {};
+let MANUALTRANSLATOR = null;
```

Comment:
Generally speaking, uppercase things in JavaScript would indicate a const. So I'd probably use `manualTranslater` here though I understand it feels very "consty" given that you only intend to update it once. Also, when using uppercase variables, use underscores to separate words: `MANUAL_TRANSLATOR`